### ISSUE TITLE
Results of ZnO linear expansion seem wrong

### ISSUE THREAD

**Issue by john-science** [terrapower/armi#886]
Look at the existing unit test for linear expansion of zinc oxide:

https://github.com/terrapower/armi/blob/f0d27e7405bde450b1ba01825c95783080974c53/armi/materials/tests/test_materials.py#L1620-L1627

Surely that is wrong?

**Comment by john-science** [comment]
@albeanth @keckler @ntouran   

Surely, that can't be right?

**Comment by keckler** [comment]
LOL

What is even going on here?? The material shrinks by 10,000%, and we test it TWICE in a row just to be certain! 🤣

**Comment by keckler** [comment]
I think clearly the correlation for `linearExpansionPercent` is incorrect, but it's hard to say what it should be since the source is a textbook that I don't have access to:
https://github.com/terrapower/armi/blob/f0d27e7405bde450b1ba01825c95783080974c53/armi/materials/zincOxide.py#L33-L47

My guess is that the multiplier on the first term is to the wrong power but hard to say.

**Comment by john-science** [comment]
> LOL
> 
> What is even going on here?? The material shrinks by 10,000%, and we test it TWICE in a row just to be certain! rofl

> At least it's not just me.

1. I added this test about 5 months ago, while adding a TON of material tests.  So the bug has been around that long.
2. As far as I can tell, the bug has been in place for a couple years.
3. No one downstream uses this material, that I know of. So, I think the bug hasn't hurt anyone yet.
4. I thought I opened this ticket when I created the bonkers unit test. Whoops, guess not.

> Does anyone have access to the cited source?
